File: Atributos de validación Data Annotations multiidioma/[C#]-Atributos de validación Data Annotations multiidioma/C#/LocalizeDataAnnotations/Models/PersonaCustomAttr.cs
using LocalizeDataAnnotations.Infrastructure;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace LocalizeDataAnnotations.Models
{
    public class PersonaCustomAttr : IPersona
    {
        [CustomRequired(ErrorMessageResourceName = "NombreObligatorio")]
        [CustomDisplay("Nombre")]
        public string Nombre { get; set; }

        [CustomDisplay("Apellido")]
        public string Apellido { get; set; }

        [CustomDisplay("Ciudad")]
        [CustomStringLength(20, MinimumLength = 4, ErrorMessageResourceName = "CiudadErrorLongitud")]
        public string Ciudad { get; set; }

        [CustomDisplay("Email")]
        public string Email { get; set; }
    }
}

File: Atributos de validación Data Annotations multiidioma/[VB]-Atributos de validación Data Annotations multiidioma/VB/LocalizeDataAnnotations/Models/PersonaCustomAttr.vb
Imports System.ComponentModel.DataAnnotations

Public Class PersonaCustomAttr
    Implements IPersona

    <CustomRequired(ErrorMessageResourceName:="NombreObligatorio")> _
    <CustomDisplay("Nombre")> _
    Public Property Nombre As String Implements IPersona.Nombre

    <CustomDisplay("Apellido")> _
    Public Property Apellido As String Implements IPersona.Apellido

    <CustomDisplay("Ciudad")> _
    <CustomStringLength(20, MinimumLength:=4, ErrorMessageResourceName:="CiudadErrorLongitud")> _
    Public Property Ciudad As String Implements IPersona.Ciudad

    <CustomDisplay("Email")> _
    Public Property Email As String Implements IPersona.Email

End Class
